contracts/lib/platform/EthAdapter.sol

```diff
@@ -52,8 +52,8 @@ contract EthAdapter is ERCDotFactory {
     }
 
     //Override
-    function unbond(address wallet, bytes32 specifier, uint quantity) internal {
-
+    function unbond1(address payable wallet, bytes32 specifier, uint quantity) internal {
```

Comment:
suspected namespacing issues so renamed function to unbond1 to prevent other bond function to be called correctly.

compiler kept throwing issue to unbond